.travis.yml

```diff
@@ -19,13 +40,17 @@ matrix:
         - cargo fmt --all -- --check
     - language: generic
       name: "dƒusion e2e Tests"
-      cache: 
+      cache:
         directories:
-          - $HOME/.npm
```

Comment:
Why is this Pr no longer valid?
https://github.com/gnosis/dex-services/pull/349

Reply:
Before, only saved the npm downloaded assets, because we used npm ci (which removes the cached `npm_modules` directory).

Now, we `npm install` and thus can cache the node modules. But it's a good point that in the very first stage (warm up npm cache) we should also be caching the downloaded files.

Reply:
`$HOME/.npm` gets cached by default for projects with a `project-lock.json`

Reply:
https://docs.travis-ci.com/user/caching/#npm-cache